Add popup menu

Apt2tell opened the pull request:
the following tasks were done
- add popup window
- add javascript to display popup window
- add style for the popup window

mwanawabangona commented on styles.css:
```diff
@@ -181,8 +181,8 @@ section {
 
 .work-card {
   box-sizing: border-box;
+  flex-direction: row;
   display: flex;
-  flex-direction: column;
   align-items: center;
```
- [ ] `required`
- [ ] Good job with the pop cards so far 👏 👏 👏 
- [ ] However the pop cards have to follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (`font`, `colours`, `images`, `text`, `margins` and the `background blur`).
- [ ] please make adjustments in accordance to the figma template on all the `cards`

`figma template`
![ya new](https://user-images.githubusercontent.com/66230306/182000332-07af7681-6229-48ea-a63b-dc620eaeff5f.png)

`your design`
![funni](https://user-images.githubusercontent.com/66230306/182000746-117383fa-7b56-4db9-8b5c-ac5ecf69ed3c.png)

mwanawabangona commented on index.html:
```diff
@@ -86,120 +88,48 @@ <h1 class="title">I’m Okechukwu<br>Glad to see you!</h1>
-    <!--middle section-->
-
-    <section class="work-container">
-        <div class="work-card" id="portfolio">
-            <img 
-                src="./images/SnapshootPortfolio.png" 
-                alt="the snapshot backshoot background one" 
-                class="snapshot-img">
+    <!-- here goes the work section  -->
+    
 
-            <div class="second-block"> 
-                <div class="primary-text">
-                    <h1 class="work-title">Tonic</h1>
-                    <ul class="work-frame">
-                        <li class="frame-item1">CANOPY</li>
-                        <li class="frame-item2">Back End Dev</li>
-                        <li class="frame-item3">2015</li>
-                    </ul>
-                    <p class="message">daily section of privately personalized reads; no accounts or sign-ups required</p>
-                </div>
-                <div class="tags">
-                    <button class="button-html">html</button>
-                    <button class="button-css">css</button>
```
- [ ] `required`
- [ ] The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).
- [ ] please make sure the pop cards have `space` between them and the original background

`figma template`
![popup-card separation](https://user-images.githubusercontent.com/66230306/182001726-6d2ef481-887b-43dd-9d9c-6d729912807d.png)

`your template`
![image](https://user-images.githubusercontent.com/66230306/182001808-45c9f07b-bf69-4ff2-b9e8-7cf959f87da5.png)

mwanawabangona reviewed (changes requested):
Hi @Apt2tell ,

Good job so far!
![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
going through all the project requirement step by step before a PR
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lameck1 reviewed (changes requested):
Hi @Apt2tell, 

Good job so far!
You've really put in a commendable effort. Just a little more needs to be done and all will be good with this milestone.

## Highlights
- Professional README ✔️
- Good commit messages ✔️ 
- Proper usage of GitHub flow ✔️ 

## Required Changes ♻️
- [ ] Please be encouraged and consider working on the concerns from the previous review.

## Optional suggestions
_N/A._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lameck1 commented on styles.css:
```diff
@@ -753,18 +923,81 @@ textarea {
-  background: #396df2;
-  color: #fff;
-}
+  .modal {
+    margin: auto;
+    width: 85%;
+    overflow: scroll;
+    background-color: rgba(0, 0, 0, 0.4);
```
You have made a lot of improvements to the modal. We have a few issues to sort out:

- [ ] On page load, the modal pop up with no content as below. This is a bug and it results in a bad user experience. Please investigate whether it can be solved.
 
  ![modal_on_load](https://user-images.githubusercontent.com/45002654/182325741-001a9ae5-35c6-4c38-9570-3f2e2154d196.png)

- [ ] We also need to refactor the style of the modal. Right now it is off the center of the screen. Please ensure the popup window is centered on the screen.

    | THE DESIGN | YOUR IMPLEMENTATION |
    | --- | --- |
    | ![image](https://user-images.githubusercontent.com/45002654/182329030-54185db7-b2f9-49db-9898-87ebe766d445.png) | ![modal](https://user-images.githubusercontent.com/45002654/182329128-8dffc9e5-a057-41ba-a4ef-510196d523b1.png) |

- [ ] Please add some margin between the `See Live` and `See Source` buttons. Also, align these buttons to the right as in the design.

  ![Screenshot 2022-08-02 112326](https://user-images.githubusercontent.com/45002654/182328582-0fdc1d74-17ef-486f-b4fe-cc859166581e.png)

- [ ] Lastly, consider reducing the size of the image in the popup. Let it be the full length of the popup window but half the height of the popup window.

Lameck1 commented on styles.css:
```diff
@@ -661,6 +828,8 @@ textarea {
   }
 
   .work-container {
+    display: flex;
+    flex-direction: column;
     margin-top: 30px;
     width: 90vw;
     gap: 88px;
```
- [ ] We can also make some improvements to the works section. Please refactor the padding in the highlighted areas in the image below. Let the `work-card` take the whole width and height with reasonable padding so as to match the design.

    | DESIGN | IMPLEMENTATION |
    | --- | --- |
    | ![image](https://user-images.githubusercontent.com/45002654/182331737-a245411f-48c8-49a7-8eea-2adff2954c5a.png) | ![Screenshot 2022-08-02 104956](https://user-images.githubusercontent.com/45002654/182330901-152e5619-23cc-4047-9da1-01007068ce17.png) |

Lameck1 reviewed (changes requested):
Hi @Apt2tell, 

You've done well addressing the concerns from the previous review 👏.
Let's consider a few areas that we still need to improve on and we'll be good to go.

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 
- GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_NA_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Apt2tell commented:
Hello sir,

I have effected the required changes.

https://github.com/Apt2tell/Okechukwu-Portfolio/pull/14

Thank you

On Tue, 2 Aug 2022 at 09:44, Lameck Otieno ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @Apt2tell <https://github.com/Apt2tell>,
>
> You've done well addressing the concerns from the previous review 👏.
> Let's consider a few areas that we still need to improve on and we'll be
> good to go.
> Highlights
>
>    - Professional README ✔️
>    - Clean code ✔️
>    - Good commit messages ✔️
>    - GitHub flow ✔️
>
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *NA*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> *Please, remember to tag me in your question so I can receive the
> notification.*
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In styles.css
> <https://github.com/Apt2tell/Okechukwu-Portfolio/pull/14#discussion_r935267550>
> :
>
> > +  .modal {
>
> +    margin: auto;
>
> +    width: 85%;
>
> +    overflow: scroll;
>
> +    background-color: rgba(0, 0, 0, 0.4);
>
>
> You have made a lot of improvements to the modal. We have a few issues to
> sort out:
>
>    -
>
>    On page load, the modal pop up with no content as below. This is a bug
>    and it results in a bad user experience. Please investigate whether it can
>    be solved.
>
>    [image: modal_on_load]
>    <https://user-images.githubusercontent.com/45002654/182325741-001a9ae5-35c6-4c38-9570-3f2e2154d196.png>
>    -
>
>    We also need to refactor the style of the modal. Right now it is off
>    the center of the screen. Please ensure the popup window is centered on the
>    screen.
>    THE DESIGN YOUR IMPLEMENTATION
>    [image: image]
>    <https://user-images.githubusercontent.com/45002654/182329030-54185db7-b2f9-49db-9898-87ebe766d445.png> [image:
>    modal]
>    <https://user-images.githubusercontent.com/45002654/182329128-8dffc9e5-a057-41ba-a4ef-510196d523b1.png>
>    -
>
>    Please add some margin between the See Live and See Source buttons.
>    Also, align these buttons to the right as in the design.
>
>    [image: Screenshot 2022-08-02 112326]
>    <https://user-images.githubusercontent.com/45002654/182328582-0fdc1d74-17ef-486f-b4fe-cc859166581e.png>
>    -
>
>    Lastly, consider reducing the size of the image in the popup. Let it
>    be the full length of the popup window but half the height of the popup
>    window.
>
> ------------------------------
>
> In styles.css
> <https://github.com/Apt2tell/Okechukwu-Portfolio/pull/14#discussion_r935278970>
> :
>
> >    .work-container {
>
> +    display: flex;
>
> +    flex-direction: column;
>
>      margin-top: 30px;
>
>      width: 90vw;
>
>      gap: 88px;
>
>
>
>    -
>
>    We can also make some improvements to the works section. Please
>    refactor the padding in the highlighted areas in the image below. Let the
>    work-card take the whole width and height with reasonable padding so
>    as to match the design.
>    DESIGN IMPLEMENTATION
>    [image: image]
>    <https://user-images.githubusercontent.com/45002654/182331737-a245411f-48c8-49a7-8eea-2adff2954c5a.png> [image:
>    Screenshot 2022-08-02 104956]
>    <https://user-images.githubusercontent.com/45002654/182330901-152e5619-23cc-4047-9da1-01007068ce17.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Apt2tell/Okechukwu-Portfolio/pull/14#pullrequestreview-1058331974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXLZMDXNFONZD4CXDPYMXDTVXDNYFANCNFSM55DZY6RA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
